config/environments/development.rb

```diff
@@ -45,4 +45,6 @@
 
   # The page to create a ticket is open to the world by default?
   config.ticket_creation_is_open_to_the_world_by_default = true
+
+  config.tenant_default_stylesheet_url = nil
```

Comment:
Our users don't like to modify our upstream source code, because it might cause conflicts when we release new versions.